static/css/thing.css

```diff
@@ -277,12 +308,12 @@
   background-image: url('../images/smart-plug-off.svg');
 }
 
-.on .smart-plug .thing-icon, .smart-plug-container.on .thing-icon {
+.smart-plug.on .thing-icon, .smart-plug-container.on .thing-icon {
```

Comment:
cool, this is a more efficient selector. There was some reason this wouldn't work before.